benchmarks/README.md

```diff
@@ -1,4 +1,19 @@
-These are the benchmarks used in testing the different WebSocket server implementations. I use WebSocket++ as clients, since that is the fastest client library that I know of, and since you can run multiple instances of the client, it shouldn't really matter what client you use as long as the server is being stressed fully.
+# uWebSockets Benchmarks
```

Comment:
I would make the caption "Benchmarks" as these are applicable to any WS implementation.